src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -279,16 +283,17 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
     }
 
     function _registerOperatorWithCoordinator(address operator, bytes calldata quorumNumbers, BN254.G1Point memory pubkey, string memory socket) internal {
-        require(
-            slasher.contractCanSlashOperatorUntilBlock(operator, address(serviceManager)) == type(uint32).max,
-            "StakeRegistry._registerOperator: operator must be opted into slashing by the serviceManager"
-        );
+        // require(
+        //     slasher.contractCanSlashOperatorUntilBlock(operator, address(serviceManager)) == type(uint32).max,
+        //     "StakeRegistry._registerOperator: operator must be opted into slashing by the serviceManager"
+        // );
```

Comment:
dangerous commenting-out, to reverse prior to merging

Reply:
actually planning to keep this out for M2 i think...

Reply:
also just going to add: I feel like this is a major decision if we're making it, not something to be done lightly without discussion of the ramifications

Reply:
Discussed offline, leaving out for now, but plan to have some form of opt-in to Slasher so we can support a smoother "migration" experience if we need to.